pyte/graphics.py

```diff
@@ -110,7 +110,7 @@
 
 #: A table of 256 foreground or background colors.
 # The following code is part of the Pygments project (BSD licensed).
-FG_BG_256 = [
+_FG_BG_256 = [
```

Comment:
This is a breaking API change, please revert.

Reply:
This shouldn't be a breaking change, this variable is redefined below as a list of strings.  I had to change this declaration to make the initial value (list of tuple[int]) a separate variable.

Reply:
Sorry, yes, I've missed that.